From ad963e921d28b0278268eb2151b3e1d45965f25e Mon Sep 17 00:00:00 2001
From: Jonathan Bakker <xc-racer2@live.ca>
Date: Mon, 6 Aug 2018 13:08:40 -0700
Subject: [PATCH] Add BOARD_BLUEDROID_VENDOR_CONF flag

Only AOSP-supported devices have hardcoded configs here, add flag to allow custom

Change-Id: Ia0135dbdfb1d13d3a3dbe0d5b26f0229df256a9e
---
 vnd_buildcfg.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/vnd_buildcfg.mk b/vnd_buildcfg.mk
index b65033a..9aa57f0 100644
--- a/vnd_buildcfg.mk
+++ b/vnd_buildcfg.mk
@@ -1,10 +1,14 @@
 intermediates := $(local-intermediates-dir)
 
+ifneq ($(BOARD_BLUEDROID_VENDOR_CONF),)
+SRC := $(BOARD_BLUEDROID_VENDOR_CONF)
+else
 SRC := $(call my-dir)/include/$(addprefix vnd_, $(addsuffix .txt,$(basename $(TARGET_DEVICE))))
 ifeq (,$(wildcard $(SRC)))
 # configuration file does not exist. Use default one
 SRC := $(call my-dir)/include/vnd_generic.txt
 endif
+endif
 GEN := $(intermediates)/vnd_buildcfg.h
 TOOL := $(TOP_DIR)external/bluetooth/bluedroid/tools/gen-buildcfg.sh
 
-- 
2.11.0

